diagnostics-app/src/app/layout/header/header.component.it.spec.ts

```diff
@@ -32,27 +31,21 @@ describe('integration: component: header', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
     service = TestBed.inject(ThemeService);
-    button = fixture.debugElement.query(By.css('button')).nativeElement;
-  });
-
-  it('service should emit event on click', () => {
-    spyOn(service, 'toggleTheme').and.callThrough();
-    expect(component.isDarkModeActivated()).toBeTrue();
-    button.click();
-    expect(service.toggleTheme).toHaveBeenCalled();
-    expect(component.isDarkModeActivated()).toBeFalse();
   });
```

Comment:
why removing this test?

Reply:
My bad, must've been removed during rebasing. Fixed it 👍🏻